docs/code_generator.md

```diff
@@ -0,0 +1,36 @@
+## Code Generator 📜
+
```

Comment:
O titulo dessa pagina eu acho que poderia ser "CLI" ou "CLI Usage" ou até "Using the CLI"